nuxeo-runtime/nuxeo-runtime-osgi/src/main/java/org/nuxeo/osgi/application/loader/FrameworkLoader.java

```diff
@@ -281,15 +268,8 @@ public static void preprocess() {
             Class<?> klass = loader.loadClass("org.nuxeo.runtime.deployment.preprocessor.DeploymentPreprocessor");
             Method main = klass.getMethod("main", String[].class);
             main.invoke(null, new Object[] { new String[] { home.getAbsolutePath() } });
-        } catch (ClassNotFoundException e) {
-            throw new RuntimeException(e);
-        } catch (SecurityException e) {
-            throw new RuntimeException(e);
-        } catch (NoSuchMethodException e) {
-            throw new RuntimeException(e);
-        } catch (IllegalAccessException e) {
-            throw new RuntimeException(e);
-        } catch (InvocationTargetException e) {
+        } catch (ClassNotFoundException | SecurityException | NoSuchMethodException | IllegalAccessException
```

Comment:
You can use `ReflectiveOperationException` for most of those, and `SecurityException` is already a `RuntimeException` and doesn't need to be re-wrapped.